ls_pre_commit_hooks/plantuml-renderer.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+if ! command -v plantuml &>/dev/null; then
+  if command -v brew &>/dev/null; then
+    brew install plantuml
+  else
+    >&2 echo 'plantuml command not found. See https://plantuml.com/download for installation instructions.'
+    exit 1
+  fi
+fi
+
+for puml_file in "${@}"; do
+  plantuml -tsvg "${@}" -o "images"
+  git add \*.svg
```

Comment:
I dont know how to make it fail if it detect this at fleet.. I think it should fail on this `git add` right?